input/parameters/lithium-ion/electrolyte_conductivity_Capiglia1999.py

```diff
@@ -9,6 +12,23 @@ def electrolyte_conductivity_Capiglia1999(c_e):
     properties of non-aqueous electrolyte solutions for rechargeable lithium batteries.
     Journal of power sources 81 (1999): 859-862.
     .. [2] http://www.cchem.berkeley.edu/jsngrp/fortran.html
+    Parameters
+    ----------
+    c_e: :class: `pybamm.Symbol`
```

Comment:
I think technically pybamm symbols aren't fed directly into this function? This function itself is only called when evaluating a `Function` node. So really the c_e that gets fed in is a number or array